doc/conf.py

```diff
@@ -767,6 +780,23 @@ def js_tag_with_cache_busting(js):
      1),
 ]
 
+# JupyterLite config
```

Comment:
Is there a way this can be disabled by default and enabled by a flag? This could be set on CI and in release mode. I'm not sure we want to be paying the build costs locally for every developer. But since the build failed, I'm not sure exactly how large that build time cost is, so it _could_ be negligible for all I know.

Reply:
The build time cost is around ~15 seconds for me locally, which seems negligible in comparison to the rest of the doc build – I've been building with `html-noplot` at this time, so it would even lesser of a fraction with CI builds which would be longer.

But to answer whether this can be disabled, I can check for the `DEVDOCS` and `is_release_build` constants so that this is avoided on local builds, as this concern was raised from the perspective of downstream packagers when building. It could make debugging a build that's failing in CI a bit of a pain, though.

Reply:
Well, let's see how long it ends up taking first.